srcpkgs/yandex-disk/template

```diff
@@ -1,23 +1,24 @@
 # Template file for 'yandex-disk'
 pkgname=yandex-disk
-version=0.1.5.940
+version=0.1.5.978
 revision=1
 only_for_archs="i686 x86_64"
 short_desc="Console client for Yandex.Disk"
 maintainer="Alexander Mamay <alexander@mamay.su>"
-license="Propietary"
+license="Proprietary"
 homepage="https://disk.yandex.com/"
 repository="nonfree"
 restricted=yes
+nostrip_files=yandex-disk
 
 if [ "$XBPS_TARGET_MACHINE" = "x86_64" ]; then
 	debpkgid="${pkgname}_${version}_amd64.deb"
 	distfiles="http://repo.yandex.ru/yandex-disk/deb/pool/main/y/${pkgname}/${debpkgid}"
-	checksum="5cafdaef3186815960be68dc93d7635e2461efe39e903bd9d074627d7605fd46"
+	checksum="163b977068018d2fe824ddec85e6e198af56c35a3dc64ccc2ee83e771f1ce73e"
 else
 	debpkgid="${pkgname}_${version}_i386.deb"
 	distfiles="http://repo.yandex.ru/yandex-disk/deb/pool/main/y/${pkgname}/${debpkgid}"
-	checksum="6eeb6330fe230da92f6e4591635e7990815400d9e0bd51188de5eeacb5217e08"
+	checksum="11f865c2e347639cf679ab32f906f033947e4e164680a9e974a1255961dee0ed"
 fi
 
 do_extract() {
```

Comment:
You can remove cd `${wrksrc}` from `do_extract()` (you are already in `$wrksrc`).

Reply:
You can remove `vmkdir usr/share/locale` from `do_install()` and just keep:
````
vcopy usr/share/locale usr/share
````
It's enough.

Also, instead of `install -m644 -D ...` you can just use:
````
vinstall etc/bash_completion.d/yandex-disk-completion.bash 644 \
	usr/share/bash-completion/completions yandex-disk
````

Reply:
Thank you very much, but I can't use vinstall, because it can't create dirs inside usr/share like bash-completion, and install -D can

Reply:
I'm pretty sure `vinstall` can automatically creates the directory, if it does not yet exist.
It's okay, I merge it anyway.